app/src/main/java/edu/grinnell/grinnell_publications_android/Services/Implementation/PublicationsRemoteClient.java

```diff
@@ -22,65 +31,110 @@
  * <p> All networking calls are made through calling on a retrofit networking service.
  * As an observable, updates to the remote calls are routed to the respective observers for either
  * changes to the UI or changes to the local data base.</p>
+ *
  * @author Albert Owusu-Asare
  * @version 1.1 Thu May  5 15:57:45 CDT 2016
  */
 public class PublicationsRemoteClient implements RemoteClientAPI {
-    private Retrofit retrofit;
-    private LocalClientAPI localClient;
-    private final PublicationsServiceAPI publicationsService;
-    private final String BASE_API = "Insert Base API URL"; //TODO: Get base URL
 
+  private Realm realm;
+  private Retrofit retrofit;
+  private LocalClientAPI localClient;
+  private AWSEndpointApiInterface api;
+  private final String BASE_API =
+      "https://g2j7qs2xs7.execute-api.us-west-2.amazonaws.com/devstable/";
 
-    public PublicationsRemoteClient(){
-        this(new RealmLocalClient());
-    }
+  public PublicationsRemoteClient() {
+    this(new RealmLocalClient());
+  }
 
-    public PublicationsRemoteClient(LocalClientAPI localClient){
-        this.localClient = localClient;
-        this.retrofit = initializeRetrofitRestAdapter();
-        this.publicationsService = this.retrofit.create(PublicationsServiceAPI.class);
-    }
+  public PublicationsRemoteClient(LocalClientAPI localClient) {
+    this.localClient = localClient;
 
-    @Override
-    public List<Publication> getAllPublications() {
-        return null;
-    }
+    retrofit = new Retrofit.Builder().baseUrl(BASE_API)
+        .addConverterFactory(GsonConverterFactory.create())
+        .build();
+    api = retrofit.create(AWSEndpointApiInterface.class);
```

Comment:
have a more descriptive name for the 'api' variable.
It's hard to tell what it represents in later parts of the code.